universe/com.goal_smart/device.js

```diff
@@ -0,0 +1,197 @@
+"use strict";
+
+const Tp = require('thingpedia');
+
+module.exports = class GoalDevice extends Tp.BaseDevice {
+  constructor(engine, state) {
+    super(engine, state);
+    this.uniqueId = 'com.goal_smart'; //
+    this.name = 'GoalSmart';
+    this.description = 'Keep up with the latest in soccer';
```

Comment:
This constructor is redundant, because the base class will fill the same information from the manifest.